### ISSUE TITLE
libusb0 driver package not added on Windows when class already exists

### ISSUE THREAD

**Issue by antoxz** [florentbr/OWON-VDS1022#104]
Is it correct for install-win.cmd to skip adding the driver package, when the ClassGUID {{EB781AAF-9C70-4523-A5DF-642A87ECA567} is present in "%SYSTEMROOT%\INF\oem*.inf"? Shouldn't it be registered anyway, to make the device recognizable by DeviceID (VID&PID)?

**Comment by Zanadar** [comment]
I just had an issue where the libusb driver was not installed by the install.cmd and the oscilloscope showed up as unknown device in device manager (windows)... manually installing the driver fixed it...

**Comment by SimonBauer-git** [comment]
yeah, i installed the driver manually from owons website, i thought you had to do that, so i guess i have circumvented that issue...
